betelgeuse.py

```diff
@@ -237,6 +237,7 @@ def add_test_case(args):
         # automated, manualonly, and notautomated
         auto_status = 'automated' if test.automated else 'notautomated'
         caseposneg = 'negative' if 'negative' in test.name else 'positive'
+        casecomponent = test.case_component if test.case_component else '-'
```

Comment:
Feel free to do the same for `caseimportance` if you want and then issue #41 can be closed.